cmds/core/unmount/unmount.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2012-2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Bind and mount modify the file name space of the current
+// process and other processes in the same name space group
+// (see fork(2)).
+
+// https://9p.io/magic/man2html/1/bind
+
+// unmount [ new ] old
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"harvey-os.org/namespace"
```

Comment:
Where is this pkg hosted? I tried to go to this URL, and normally I'd expect it to forward me to GitHub or gitlab or someplace I can look at the code and contribute.

Reply:
the package is hosted here https://github.com/harvey-os/go
I think I screwed up trying to split the code between two repos because now to change stuff, I have to update in two places. I think I should move that stuff under `pkg/namespace` or `pkg/ns`, what do you think?